bindings/serialize.h

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SERIALIZE
+#define SERIALIZE
+
+#include <memory>
+#include <map>
+#include "v8-profiler.h"
+
+std::vector<char> serializeHeapProfile(std::unique_ptr<v8::AllocationProfile>
+  profile, int64_t intervalBytes, int64_t startTimeNanos);
+
+std::vector<char> serializeTimeProfile(std::unique_ptr<v8::CpuProfile>,
```

Comment:
Why not std::vector<uint8_t> which is more like byte type?

Reply:
Nan::CopyBuffer takes in a char*, so its consistent with the type needed to make a buffer to return to node.

When I switched to uint8_t, I got this error: 
```
../bindings/sampling-heap-profiler.cc:94:45: error: invalid conversion from ‘__gnu_cxx::__alloc_traits<std::allocator<unsigned char> >::value_type* {aka unsigned char*}’ to ‘const char*’
```